anvil/src/input_handler.rs

```diff
@@ -121,6 +126,7 @@ impl<Backend> AnvilState<Backend> {
                         .get_surface_and_bring_to_top(self.pointer_location);
                     self.keyboard
                         .set_focus(under.as_ref().map(|&(ref s, _)| s), serial);
+                    self.text_input.set_focus(under.as_ref().map(|&(ref s, _)| s));
```

Comment:
What is the reason behind `|&(ref s, _)|`, from my quick testing `|(s,_)|` gives the same result and looks cleaner.

Reply:
Alright, I'll replace that, it was basically a quick copy past

Reply:
Done